index.android.js

```diff
@@ -0,0 +1,84 @@
+'use strict';
+
+var React = require('react-native');
+var {
+  AppRegistry,
+  StyleSheet,
+  Component,
+  Text,
+  View,
+  Navigator,
+  TouchableOpacity,
+} = React;
+
+var LoginScreen = require('./LoginScreen');
+var MainPage    = require('./MainPage');
+
+class AwesomeProject extends Component {
+  
+  render() {
+    return (
+
+       <Navigator
+          initialRoute={{id: 'LoginScreen', name: 'Index'}}
+          renderScene={this.renderScene.bind(this)}
```

Comment:
do you understand why you need to use bind(this) for the renderScene method of AwesomeProject?